### DISCUSSION TITLE
SuperBench version number field in confg

### DISCUSSION THREAD

**Discussion by jaredAMD** [microsoft/superbenchmark#416] [Q&A]
Can you comment on the consequence of the version number field in SuperBench configs? By way of example: what would results from specifying `0.5` in a `0.6` config? Can you point me to documentation describing this field, and how it's intended to be used now (and in future releases)?

**Comment by abuccts** [comment]
It was designed for compatibility check, e.g., throw error if trying to run a config with higher version using the cli with lower version. But this is not implemented yet and the version field is not in use as of now..
